### ISSUE TITLE
Videocall and Screen sharing is not working simultaneously in viewpager adapter's fragment.

### ISSUE THREAD

**Issue by rushdroid** [opentok/opentok-android-sdk-samples#226]
@robjperez So in my case, I do have one Activity and two fragments, 1st is for video call and 2nd is for screen share. I have taken code reference from this git repo for both video and screen share.
My code structure is.

MainActivity.java
     -> VideoFragment.java
     -> ScreenShareFragment.java
Both fragment is loaded via viewpager.

So now what happens here when I start screen share and video call simultaneously, Video call opponent vide override by screen share and screen share goes black/blank.

So basically i got screenshare callbacks on my videoview and screenshare view both.

@robjperez  Can you help me on this?

Thanks

**Comment by rushdroid** [comment]
@robjperez Any update on this?

**Comment by rushdroid** [comment]
@robjperez Any update on this?

**Comment by rushdroid** [comment]
There is no support by this GitHub repo owner.

**Comment by rperezcubero** [comment]
I'm not really sure what the problem is :(

As I read your use case I would say you need two publishers, one for the video and the other for the screen sharing.

Not sure if you are already doing this.

Going to close the issue since it is very old. Please reopen it if necessary
